src/map/SemanticMarkers.tsx

```diff
@@ -78,7 +80,7 @@ export default function SemanticMarkers({ onViewportChanged, markers, animation}
       timeoutVariable = setTimeout(
           () => {
             setConsolidatedMarkers([...markers])
-          }, animation ? animation.duration : 0
+          }, duration
```

Comment:
Will getting rid of this conditional statement mean that there will be a timeout even if an animation isn't happening?

Reply:
Oh, good point.  I shouldn't have removed the conditional...